tests_e2e/test_suites/agent_not_provisioned.yml

```diff
@@ -0,0 +1,12 @@
+#
+# Disables Agent provisioning using osProfile.linuxConfiguration.provisionVMAgent and verifies that the agent is disabled
+# and extension operations are not allowed.
+#
+name: "AgentNotProvisioned"
+tests:
+  - "agent_not_provisioned/agent_not_provisioned.py"
+images: "ubuntu_2004"
```

Comment:
SSH is not being enabled correctly in a few distros when the agent is not provisioned. A quick check gave me the impression it may be an issue with cloud-init. Restricting this suite to Ubuntu 20 while I investigate, all make it random after.